client/src/components/dashboardPage/Dashboard.js

```diff
@@ -1,177 +1,213 @@
   },
-  cardSectionLabels: {
-    margin: 0,
-    padding: 0
+  table: {
+    background: theme.palette.secondary.main,
+    [theme.breakpoints.only('sm')]: {
+      width: '60vw'
+    },
+    [theme.breakpoints.only('xs')]: {
+      width: '90vw'
+    },
+    maxWidth: 1000,
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    justifyContent: 'space-around',
+    alignItems: 'center'
   },
-  classCard: {
+
+  tableHead: {
+    width: '100%',
     display: 'flex',
-    flexDirection: 'column',
-    margin: '1rem',
-    background: 'white',
-    width: '200px',
-    textDecoration: 'none',
-    color: theme.palette.secondary.contrastText
+    flexFlow: 'column nowrap',
+    alignItems: 'center'
   },
-  cardTitle: {
-    color: theme.palette.secondary.contrastText,
-    textAlign: 'center',
-    fontSize: '1rem'
+  tableBody: {
+    width: '100%',
+    dislay: 'flex',
+    flexFlow: 'column nowrap',
+    alignItems: 'center'
   },
-  card: {
+  tableRow: {
     display: 'flex',
-    flexFlow: 'column nowrap',
+    flexFlow: 'row nowrap',
+    width: '100%',
     justifyContent: 'center',
     alignItems: 'center',
-    margin: '5% 0',
-    background: theme.palette.secondary.main,
-    width: 125,
-    height: 125,
-    color: theme.palette.secondary.contrastText,
-    [theme.breakpoints.down('sm')]: {
-      margin: '10% 0'
-    },
+    fontSize: '1.6rem'
+  },
+  tableCell: {
+    fontSize: '1rem',
+    width: '33%',
+    color: theme.palette.primary.dark
+  },
+  bodyText: {
+    textAlign: 'center'
+  },
+  aLink: {
+    textDecoration: 'none',
+    fontSize: '1rem',
+    color: theme.palette.primary.dark,
     '&:hover': {
-      background: theme.palette.secondary.dark
+      cursor: 'pointer',
+      color: theme.palette.primary.light
     }
-  },
+  }
 });
 
 const Dashboard = props => {
-  // const name = localStorage.getItem('name'); // commented out until decide what to do w/ name
-  const {
-    userClasses,
-    classes,
-    userRefreshrs,
-    getClasses,
-    getRefreshrs
-  } = props;
+  const { classes, id, token, history } = props;
+  const [campaigns, setCampaigns] = useState([]);
+  const [name, setName] = useState('');
+
+  //STATS
+  const userCampaigns = (id, token) => {
+    console.log(id);
+    axios({
+      method: 'get',
+      url: `http://localhost:9000/campaigns/user/${id}`,
+
+      //url: `https://refreshr.herokuapp.com/campaigns`,
+      headers: { Authorization: `Bearer ${token}` }
+    })
+      .then(res => {
+        //console.log(res);
+        setCampaigns(res.data.campaigns);
+      })
+      .catch(err => console.log(err));
+  };
+  const createData = (id, classname, preview, date, classID) => {
+    return { id, classname, preview, date, classID };
+  };
+
+  const rows = [];
+
+  campaigns.forEach(c => {
+    console.log(c);
+    const current = new Date();
+    if( current === moment(c.date).subtract(30, 'days')){
+        const date = moment(c.date).format('Do/MMM/YYYY')
```

Comment:
I did it this way so that the app was more inclusive as most other parts of the world write dates as d/m/y, so I was trying to avoid being American-centric

Reply:
Ah, okay, that makes sense. Maybe DD/M/YYYY.